payment_allpay/__openerp__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+{
+    'name': 'allPay Payment Acquirer',
+    'category': 'Website',
+    'summary': 'Payment Acquirer: allPay Implementation',
+    'version': '1.0',
+    'description': """allPay Payment Acquirer""",
+    'author': 'Odoo CN, Jeffery <jeffery9@gmail.com>',
```

Comment:
follow https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py and add Odoo Community Association as Author